rules/xcodeproj.bzl

```diff
@@ -252,7 +253,22 @@ def _exclude_swift_incompatible_define(define):
         return token
     return None
 
-def _joined_header_search_paths(hmap_paths):
+def _fw_search_paths_for_target(target_name, all_transitive_targets):
+    # Ensure Xcode will resolve references to the XCTest framework.
+    framework_search_paths = ["$(PLATFORM_DIR)/Developer/Library/Frameworks"]
+
+    # all_transitive_targets includes all the targets built with their different configurations.
+    # Some configurations are only applied when the target is reached transitively
```

Comment:
`reached transitively`.... this is so new to me :)

Reply:
i feel like we should start using subway metaphor, ie: two station can be connected directly or transitively ;)

Reply:
lol! It has to do with traversal from on target to another through the dependency graph. I hoped this explanation was clear but feel free to suggest a re-wording :)